less/index.less

```diff
@@ -54,6 +54,259 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
 }
 
+
+
+html {
+	font-size: 62.5%;
+	
+}
+h1 {
+	font-size: 6rem;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+body {
+	font-size: 1.6rem;
+	background-color: @pastyColor;
+	font-family: @balooFont;
+}
+
+@pastyColor: rgb(236, 233, 216);
+@balooFont: 'Ubuntu Condensed', sans-serif;
+
+.fDisplay(){
```

Comment:
as a general observation, using a mixin for a single property is probably not worth it, as it doesn't save you anything and makes it less obvious what is going on in your code than if you'd just used the underlying property